.github/workflows/pr-checks.yml

```diff
@@ -34,3 +34,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: echo ${{ matrix.directory }}
+
+  terraform-plan:
+    name: Terraform Plan
+    needs: terraform-changes
+    strategy:
+      matrix:
+        directory: ${{ fromJSON(needs.terraform-changes.outputs.directories).matrix }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hashicorp/setup-terraform@v2
+      - env:
+          AWS_DEFAULT_REGION: "eu-west-2"
+          GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
```

Comment:
Can it use the ${{ secrets.GITHUB_TOKEN }} built in token here?

Reply:
In this case it’s accessing the analytical services org, so default token won’t have access, it will need a PAT

Reply:
I just had a quick read, and yes! That sounds like it'll work much better! 🥳 

Can take out my PAT token which felt a bit ewy to generate 🤢

Reply:
and thanks @jacobwoffenden for saving me the time of figuring that out! 😂

Reply:
Tru the bult in one is for the repo, to do org stuff will need the operations-engineering / org level token

Reply:
I’d recommend the approach we have in Data Platform, and use a token generated from a robot account (@moj-data-platform-robot), that way they aren’t _personal_ PATs

Reply:
That's the plan; I just did a personal one last night to test it out 🧪 👀 but it expires in 30 days, so it was never a long-term solution 😂

Reply:
True we got the ops-eng bot for that